### ISSUE TITLE
[FEATURE] Download to Remote Server - FTP

### ISSUE THREAD

**Issue by Vallesuski** [robvdpol/RaceControl#275]
Current download function is for a local machine.
It would be great if you can add a 2nd download option to a remote server via ftp.

Let's say you press "download button" and then ask you to download into a "local computer" or "remote server". If remote server is selected then ask for ftp credentials (hostname, username & password) and put the file into the server.

This function would be really great for those who owns remote servers with 1gbps/10gbps connections, so they can take advantage of the speed. Another benefit is to "stream" downloaded files directly from remote server being in your parents or friends house.

**Comment by TTCashier** [comment]
Would love this also.

**Comment by robvdpol** [comment]
I don't really see how this would work technically. If you upload the stream via FTP you're not getting the benefit of the remote server bandwidth, the data would pass through Race Control to the FTP server. You'd be better off simply running Race Control on the remote server I think?

**Comment by Vallesuski** [comment]
Most likely all remote servers are Linux based, so I don't see how we could run RaceControl directly on the server.

When I press "download" currently on Windows version, it downloads from F1 server to local computer using ffmpeg? Or there is other job doing the download?

**Comment by robvdpol** [comment]
The download is performed by Flyleaf library, which indeed uses FFMpeg.

For Linux you might be better off using [f1viewer](https://github.com/SoMuchForSubtlety/f1viewer).

**Comment by Vallesuski** [comment]
Yes, I'm aware of f1viewer. I use both: RaceControl and f1viewer.

I think we should close this feature.
